meetings/2024-03-06.md

```diff
@@ -45,12 +50,93 @@ URL: https://github.com/solid/specification/discussions/618
+* PAC: TBL and I are saying the same thing I think we change the charter and for the Council to decide any way. Please adjust and then you can go. Truly this is not TBL's recommendation but ???. I still have some changes to push to GitHub and submit to the CG. I'm currently re-writing the Mission. So, TBL, this WIP.
+* TBL: Ok, thanks.
+* eP: I do support some natural language on read-write protected data. Nasty contrast with open data. So here is more protected resources. I participated in the FedCM WG proposal. There is possible overlap. There is no problem with it but Solid WG was pointe out. Especially with things like Solid-OIDC. If FedCM WG goes ahead then we might have alternative to Solid-OIDC. We should at least consider listing them as a possible Group to coordinate with.
+* HZ: Do you still want to do the poll for changing the name?
+* SC: Thinking about the name depends on the scope/mission. What do you want to get out of this poll 
+* PAC: Whether people think it is important to stick to "Solid" or not. I agree that the name depends on the mission. At this point do we feel we should stick with Solid or change.
+* HZ: I'm neutral but biased towards keeping the name because people are familiar with it. I don't know if you find a name that reflects.
+* SC: I don't have a strong opinion, I don't think it matters. In SocialWG, there was ActivityPub, AS2, LDN etc. Similarly this WG could publish a set of spec. In SocialWG some specs were overlaping. I don't think that the name has branding value to the working group. At some point it will be about sucess of the spec. I'm open to the idea to pick a different name. I think about Socially aware web, or talking about fixing the Web. I'm open to changing the name. Depends on the scope/mission/deliverables.
+* AC: I'm interested in the scope/mission. Content coming out of the WG. If it helps us move the WG along, then happy to move. Basically neutral. Leaning slightly on changing the name.
+* TBL: In principle on changing the name, as mentioned before with read-write protected/authenticated data. Even discussing in this group we have different opinions.
+* eP: Sometimes branding can create tension especially in standards. To accomplish the mission of Solid, we need something more than what this Working Group can accomplish. Solid could push a storage spec, access control spec, etc. that other things approaches can also work with.. Thre are other projects in the same perimiter and don't worry about naming them because the Solid will build on them. There is more to Solid.
```

Comment:
```suggestion
* TBL: In principle on changing the name, as mentioned before with read-write protected/authenticated data. Even discussing in this group, we have different opinions.
* eP: Sometimes branding can create tension, especially in standards. To accomplish the mission of Solid, we need something more than what this Working Group can accomplish. Solid could push a storage spec, access control spec, etc., that other things can also work with. There are other projects in the same perimeter; don't worry about naming them because Solid will build on them. There is more to Solid.
```